examples/colab_utils/colab_install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#Don't run again if it's already installed
+[ -f /content/habitat_sim_installed ] && exit
+
+#Install Miniconda
+cd /content/
+wget -c https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && bash Miniconda3-latest-Linux-x86_64.sh -bfp /usr/local
+
+##Install Habitat-Sim and Magnum binaries
+conda config --set default_threads 4 #Enables multithread conda installation
+conda install -y --prefix /usr/local -c aihabitat -c conda-forge habitat-sim headless withbullet python=3.6
```

Comment:
https://towardsdatascience.com/conda-google-colab-75f7c867a522 it's the path of least resistance to get this working properly.

Reply:
@dhruvbatra Actually, thank you so much for that as I just got inspired to a much hackier way to install conda with symlinks that removes the need to do ANY sys.path hacking! It's definitely not the best idea for a longterm install since pip and conda are installed in the same exact place, but it works!